### PR TITLE
GH-2159: Update frequency reduction, AB test opt-out, remove Cliqz AB test

### PR DESCRIPTION
**Pull request by wlycdgr**
* Reduces frequency of CMP, bug DB, and AB test check from once an hour to once a day
* Adds new opt-out setting for AB test participation in Settings -> Opt In/Out
* Removes Cliqz antitracking AB test (`background.js` code refactored and simplified accordingly)
* Adds `ts` query string param to Metrics pings to signify whether user has AB tests enabled
* Disables sending other AB tests query string params if the user has AB tests disabled

* [X] Have you followed the guidelines in [CONTRIBUTING.md](https://github.com/ghostery/ghostery-extension/blob/master/CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Have you added an explanation of what your changes do?
* [X] Does your submission pass tests?
* [X] Did you lint your code prior to submission?

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -1078,38 +1088,6 @@ function onMessageHandler(request, sender, callback) {
...
 }
 
-/**
- * Determine Antitracking configuration parameters based
- * on the results returned from the abtest endpoint.
- * @memberOf Background
- *
- * @return {Object} 	Antitracking configuration parameters
- */
-function getAntitrackingTestConfig() {
```
Comment (christophertino) [inline src/background.js:1088]: @sammacbeth Can you review this section to make sure we've got the anti-track setting correct?

**Comment by christophertino** [comment]
Addresses the AB Test opt-out for #561

File: src/background.js [hunk 2]
```diff
@@ -1129,29 +1107,48 @@ function setupHubLayoutABTest() {
...
-		}
-		Object.keys(antitrackingConfig).forEach((opt) => {
-			const val = antitrackingConfig[opt];
-			log('antitracking', 'set config option', opt, val);
-			antitracking.action('setConfigOption', opt, val);
-		});
-	}
+function setupBloomFilterABTest() {
 	if (abtest.hasTest('antitracking_whitelist2')) {
 		cliqz.prefs.set('attrackBloomFilter', false);
```
Comment (sammacbeth) [inline src/background.js:1117]: We don't use this setting anymore. This test can be removed.

**Review by sammacbeth — CHANGES_REQUESTED** [review]
Anti-tracking configuration looks good. Just the ABtest is no longer used so you can remove that code.

File: src/background.js (hunk 2 above)
Comment (wlycdgr) [reply src/background.js]: Done

File: src/background.js [hunk 3]
```diff
@@ -1706,11 +1694,11 @@ function initializeGhosteryModules() {
 			// auto-fetch from CMP
 			cmp.fetchCMPData();
 
-			if (!IS_CLIQZ) {
+			if (!IS_CLIQZ && conf.enable_abtests) {
 				// auto-fetch human web offer
```
Comment (christophertino) [inline src/background.js:1698]: i don't think this comment applies anymore

**Review by christophertino — APPROVED** [review]
Looks good

**Pull request merged**